packages/protocol/test/identity/identityproxy.ts

```diff
@@ -0,0 +1,46 @@
+import { assertRevert } from '@celo/protocol/lib/test-utils'
+import {
+  IdentityProxyContract,
+  IdentityProxyInstance,
+  IdentityProxyTestContract,
+  IdentityProxyTestInstance,
+} from 'types'
+
+const IdentityProxy: IdentityProxyContract = artifacts.require('IdentityProxy')
+const IdentityProxyTest: IdentityProxyTestContract = artifacts.require('IdentityProxyTest')
+
+contract('IdentityProxyHub', (accounts: string[]) => {
```

Comment:
I notice this does not test sending calls with `value` 👀  Is that a feature we are not supporting here?